compiler/visq/visq

```diff
@@ -107,6 +123,64 @@ def _run_dalgona(model, data, analysis, save_dir):
         raise
 
 
+def save_inputs(inputs, inputs_path):
+    # Create h5 file
+    output_path = inputs_path + "/inputs.h5"
+    h5_file = h5.File(output_path, 'w')
+    group = h5_file.create_group("value")
+    group.attrs['desc'] = "Input data"
+
+    for i in range(len(inputs)):
+        sample = group.create_group(str(i))
+        for j in range(len(inputs[i])):
+            sample.create_dataset(str(j), data=inputs[i][j])
+
+    h5_file.close()
+    return output_path
+
+
+def advance_on_data(models, data_batch_path, computers):
+
+    curr_dir = Path(__file__).parent.resolve()
+    dump_fp32_py = curr_dir / 'visqlib' / 'DumpFP32FM.py'
+    dump_fq_py = curr_dir / 'visqlib' / 'DumpFakeQuantFM.py'
+
+    with tempfile.TemporaryDirectory() as fp32_dir, \
+         tempfile.TemporaryDirectory() as fq_dir:
+
+        #print("advance on data + {fp_dir} + {fq_dir}".format(fp_dir = fp32_dir, fq_dir = fq_dir))
+        _run_dalgona(models[0], data_batch_path, dump_fp32_py, fp32_dir)
+        copyfile(fp32_dir + '/tensors.txt', fq_dir + '/tensors.txt')
+        _run_dalgona(models[1], data_batch_path, dump_fq_py, fq_dir)
+
+        for metric_key in computers:
+            computers[metric_key][0].advance_on(fp32_dir, fq_dir)
+
+
+def _run_simultaneously(models, data, computers, batch_size):
```

Comment:
The name looks a bit confusing. Maybe `_run_batch` or something else would be better.

Reply:
> The name looks a bit confusing. Maybe `_run_batch` or something else would be better.

ok